### PR TITLE
fix: gamification feedback

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes

- Describe what this PR does
- top tags space vertically is too big
- top tags is missing 1
- userReadingRankHIstory is offset incorrectly
- max width modal 420px
- links profile should open a new tab

## Manual Testing

- On those affected packages:
- [x] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?

DD-{number} #done

### CODE REVIEW

File: packages/shared/src/components/profile/ActivitySection.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export const ActivitySectionHeader = ({
         {title}
         <ActivitySectionSubTitle>
           {subtitle}
-          <ClickableText tag="a" className="ml-1" href={link}>
+          <ClickableText tag="a" target="_blank" className="ml-1" href={link}>
```
Comment (rebelchris) [inline packages/shared/src/components/profile/ActivitySection.tsx:45]: I think there should be two buttons here right?

**Review by rebelchris — APPROVED** [review]
There should be two links on the profile page, otherwise good work!

File: packages/shared/src/components/profile/ActivitySection.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/profile/ActivitySection.tsx:45]: This is a reused component so both of them do this now.

**Comment by sshanzel** [comment]
Preview:

Screen Shot 2022-02-03 at 10 06 23 PM

**Review by idoshamun — APPROVED** [review]
Go go go go!

**Pull request merged**